client/src/components/molecules/NetworkSwitcher.module.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../styles/variables';
+
+.networkListWrapper {
+    position: relative;
+    cursor: pointer;
+    font-size: $font-size-mini;
+}
+
+.networkListWrapper em {
+    display: inline-block;
+    border: 1px solid $brand-grey-lighter;
+    padding: 4px;
```

Comment:
we try to only use `rem` units for all spacing, and a common way to get consistent spacing is to work with the `$spacer` variable, like `padding: $spacer / 4 $spacer / 2`

Reply:
done